app/views/articles/show.html.erb

```diff
@@ -63,9 +65,10 @@
 <div class="l-nav l-article-3col-right">
   <%= render 'shared/want_to_talk' %>
   <div class='related-links--desktop' data-dough-component="StickyColumn">
-
-    <% unless @campaign_promo_excluded %>
-      <%= render 'articles/survive_january_promo' %>
+    <% if display_campaign_promo? %>
```

Comment:
Wrapping a condition around another condition may not be ideal, nonetheless this is temporary and due to be destroyed by the end of January